### PR TITLE
[OSPRH-6762] Logging does not have singular variant of the resource

### PR DESCRIPTION
**Pull request by jlarriba**

### CODE REVIEW

File: api/bases/telemetry.openstack.org_loggings.yaml [hunk 1]
```diff
@@ -9,6 +9,8 @@ metadata:
 spec:
   group: telemetry.openstack.org
   names:
+    categories:
+    - logging
```
[the remark this answers is not in the document]
Comment (jlarriba) [reply api/bases/telemetry.openstack.org_loggings.yaml:13]: I think the issue is that there are other resources who have "logging" as a category. Apparently, the CLI in most recent versions searches first for category and then for name. IMHO this is erroneus behaviour, but effectively is making "logging" a reserved word. With this search pattern, it seems that any word that is used as a category would be reserved by the resource using it.

The actual solution, as specified in the ticket, would be to rename the resource to something more specific, like "openstacklogging" or something like that. I don't really see other possibility. However, I dont know if this is a change that will go for GA, so for the moment this is the proposal.

Adding the logging category here at least allows the CLI to return something, even if it is not what we want, it is better than not returning anything.

**Review by csibbitt — APPROVED** [review]

**Comment by jlarriba** [comment]
recheck

**Comment by jlarriba** [comment]
recheck

**Pull request merged**